docs/source/models.rst

```diff
@@ -1247,6 +1247,61 @@ models to be interpreted as generic Python functions for inference via
 You can also use the :py:func:`mlflow.prophet.load_model()`
 method to load MLflow Models with the ``prophet`` model flavor in native prophet format.
 
+Prophet pyfunc usage
+~~~~~~~~~~~~~~~~~~~~
+
+This example uses the `example_wp_log_peyton_manning.csv <https://github.com/facebook/prophet/blob/main/examples/example_wp_log_peyton_manning.csv>` _
+from Prophet's GitHub repository. The dataset looks like this:
+
+============= =================
+ds            y
+============= =================
+2007-12-10    9.59076113897809
+2007-12-11    8.51959031601596
+2007-12-12    8.18367658262066
+2007-12-13    8.07246736935477
+============= =================
+
+.. code-block:: py
+
+    import pandas as pd
+    from prophet import Prophet
+    import mlflow
+
+    train_df = pd.read_csv("https://raw.githubusercontent.com/facebook/prophet/main/examples/example_wp_log_peyton_manning.csv")
+    # starts on 2007-12-10, ends on 2016-01-20
+    print(f"Training data start: {train_df.ds.iloc[0]}")
+    print(f"Training data end: {train_df.ds.iloc[-1]}")
+    reg_model = Prophet()
+    reg_model.fit(train_df)
+
+    # Serialize the Model
+    with mlflow.start_run():
+        model_info = mlflow.prophet.log_model(reg_model, "prophet-model")
```

Comment:
Please extract the parameters from the Prophet model (the ones that are not Objects) and log them to the run.

Reply:
Done. Note that my change logs the following: 

```python
{'changepoint_prior_scale': 0.5,
 'changepoint_range': 0.8,
 'daily_seasonality': 'auto',
 'growth': 'linear',
 'holidays_prior_scale': 10.0,
 'interval_width': 0.8,
 'logistic_floor': False,
 'mcmc_samples': 0,
 'n_changepoints': 25,
 'seasonality_mode': 'additive',
 'seasonality_prior_scale': 10.0,
 'specified_changepoints': False,
 'uncertainty_samples': 7,
 'weekly_seasonality': 'auto',
 'y_scale': 12.846746888829,
 'yearly_seasonality': 'auto'}
```
and not the ``prophet_model.params`` dict, which is all ndarrays.